lms/djangoapps/course_api/views.py

```diff
@@ -170,4 +175,10 @@ def get_queryset(self):
         Return a list of courses visible to the user.
         """
         username = self.request.query_params.get('username', self.request.user.username)
-        return list_courses(self.request, username)
+        courses = list_courses(self.request, username)
+
+        org = self.request.query_params.get('org')
```

Comment:
I second that as i believe it is the more efficient/performant way to go as well.